src/Honeybee_Generator_Wind_Horizontialaxis.py

```diff
@@ -134,206 +134,212 @@ def checktheinputs(_name,_simpleOrAnalytical,_powerCoefficients,_powerControl,_r
         ghenv.Component.AddRuntimeMessage(w, "Only one template can be used at a time! Please set either TemplateLargeTurbine_ or TemplateMediumTurbine_ to true not both!")
         return -1
 
-    if _powerControl == None:
-        print "The field _powerControl must be specified!"
+    if powerControl_ == None:
```

Comment:
if an input is required it should be _input and empty on the right side. The older version was correct. I'm not sure why are you changing them to input_? Is powerControl an optional input?